From: Jiri Slaby <jslaby@suse.cz>
Subject: kabi: hide proc_fs.h inclusion
Patch-mainline: never

Commit 6f6f21eceec3a60d2066bb3b5e31dc14f9168fa7 (namespaces, pid_ns:
fix leakage on fork() failure), upstream 5e2bf0142231194d36fdc9596b36a261ed2b9fe7
added linux/proc_fs.h inclusion to kernel/fork.c in 3.0.32. But this
defined a bunch of symbols which causes kabi breakage now.

So hide the inclusion for the __GENKSYMS__ case.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 kernel/fork.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/kernel/fork.c
+++ b/kernel/fork.c
@@ -48,7 +48,9 @@
 #include <linux/audit.h>
 #include <linux/memcontrol.h>
 #include <linux/ftrace.h>
+#ifndef __GENKSYMS__
 #include <linux/proc_fs.h>
+#endif
 #include <linux/profile.h>
 #include <linux/rmap.h>
 #include <linux/ksm.h>
